frontend/tests/features/wheresMyGene/wheresMyGene.test.ts

```diff
@@ -21,7 +26,7 @@ const HOMO_SAPIENS_TERM_ID = "NCBITaxon:9606";
 
 const GENE_LABELS_ID = "[data-testid^=gene-label-]";
 const CELL_TYPE_LABELS_ID = "cell-type-name";
-const ADD_TISSUE_ID = "add-tissue-btn";
+const TISSUE_LABELS_ID = "tissue-name";
```

Comment:
Some minor lint issues - not sure why `npm run lint` is not capturing these?

<img width="1010" alt="Screenshot 2023-08-17 at 11 02 22 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/9dfd4809-1f2e-4f67-b069-f8296728ee2f">
<img width="1176" alt="Screenshot 2023-08-17 at 11 02 29 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/3b02dc88-5880-470a-adca-a4832d97f5ef">

Reply:
Oh right we still need to do this ticket:

https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-data-portal/5055